bin/nodeshift

```diff
@@ -145,6 +145,10 @@ yargs
   .argv;
 
 function commandHandler (argv) {
+  if (argv.quiet) {
```

Comment:
can you try something before we merge this.   When we require the common-log module,  we are also executing the function that gets imported to setup the logger.   Can you require the logger like this `const logger = require('common-log')` , you can keep it at the top,  but here,  instead of the if statement,  call `const log = logger()`

Reply:
Sure, I'm going to try that thanks.

Reply:
I just realized, that you might also have to not require "cli" at the top and require it in the commandHandler after you do the logger stuff.